components/Tile.tsx

```diff
@@ -14,26 +37,50 @@ export default function Tile({ tile }) {
                 position="relative"
                 flexWrap="wrap"
             >
-                <Badge marginLeft={0} fontSize={15}>{tile.type}</Badge>
+                <Badge color={colData} marginLeft={12} fontSize={15}>{tile.type}</Badge>
                 <Pane
                     marginTop={20}
+                    marginLeft={12}
                     display="flex"
                     flexDirection="column"
                     alignItems="flex-start"
                 >
                     <span style={{ fontSize: '20px' }}><b>{tile.title}</b></span>
                     <span style={{ fontSize: '15px' }}><b>${tile.amt}</b></span>
                     <img id="listingImage" src={tile.thumbnail} alt={tile.title} />
+                    <Pane
+                        paddingTop={15}
+                        fontSize="15px"
+                    >
+                        {tile.description}
```

Comment:
this is optional but description length can be handled with text-overflow https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow but this is not very important and can be added later.